java/com/google/javascript/jscomp/JsChecker.java

```diff
@@ -314,7 +319,11 @@ public CheckLevel level(JSError error) {
     return result.build();
   }
 
-  public static void main(String[] args) throws IOException {
+  static int processRequest(Collection<String> args) throws IOException {
+    String lastArg = Iterables.getLast(args, "");
+    if (lastArg.startsWith("@")) {
+      args = Files.readAllLines(Paths.get(lastArg.substring(1)), UTF_8);
+    }
```

Comment:
Thanks, I actually had not seen the ExampleWorker yet. I agree your code is fine as-is.